doc/using_xgboost.rst

```diff
@@ -0,0 +1,200 @@
+* Exensibility - Because the open source XGBoost container is open source,
+  you can extend the container to install additional libraries and change the version of XGBoost that the container uses.
+  For more information, see `SageMaker XGBoost Container <https://github.com/aws/sagemaker-xgboost-container>`__.
+
+
+***********************************
+Use XGBoost as a Built-in Algortihm
+***********************************
+
+Amazon SageMaker provides XGBoost as a built-in algorithm that you can use like other built-in algorithms.
+Using the built-in algorithm version of XGBoost is simpler than using the open source version, because you don't have to write a training script.
+If you don't need the features and flexibility of open source XGBoost, consider using the built-in version.
+For information about using the Amazon SageMaker XGBoost built-in algorithm, see `XGBoost Algorithm <https://docs.aws.amazon.com/sagemaker/latest/dg/xgboost.html>`__
+in the *Amazon SageMaker Developer Guide*.
+
+*************************************
+Use the Open Source XGBoost Algorithm
+*************************************
+
+If you want the flexibility and additional features that it provides, use the SageMaker open source XGBoost algorithm.
+
+For a complete example of using the open source XGBoost algorithm, see the sample notebook at
+https://github.com/awslabs/amazon-sagemaker-examples/blob/master/introduction_to_amazon_algorithms/xgboost_abalone/xgboost_abalone_dist_script_mode.ipynb.
+
+
+Train a Model with Open Source XGBoost
+======================================
+
+To train a model by using the Amazon SageMaker open source XGBoost algorithm:
+
+.. |create xgboost estimator| replace:: Create a ``sagemaker.xgboost.XGBoost estimator``
+.. _create xgboost estimator: #create-an-estimator
+
+.. |call fit| replace:: Call the estimator's ``fit`` method
+.. _call fit: #call-the-fit-method
+
+1. `Prepare a training script <#prepare-a-training-script>`_
+2. |create xgboost estimator|_
+3. |call fit|_
+
+Prepare a Training Script
+-------------------------
+
+A typical training script loads data from the input channels, configures training with hyperparameters, trains a model,
+and saves a model to model_dir so that it can be hosted later.
+Hyperparameters are passed to your script as arguments and can be retrieved with an argparse.ArgumentParser instance.
+
+For a complete example of an XGBoost training script, see https://github.com/awslabs/amazon-sagemaker-examples/blob/master/introduction_to_amazon_algorithms/xgboost_abalone/abalone.py.
+
+Let's look at the main elements of the script. Starting with the ``__main__`` guard,
+use a parser to read the hyperparameters passed to the estimator when creating the training job.
+These hyperparameters are made available as arguments to our input script.
+We also parse a number of Amazon SageMaker-specific environment variables to get information about the training environment,
```

Comment:
I think you could do something similar as with the other frameworks, where some of the common variables are listed, and then there's a link to the exhaustive list, e.g. https://sagemaker.readthedocs.io/en/stable/using_mxnet.html#for-versions-1-3-and-higher